browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -3929,13 +3952,18 @@ L.CanvasTileLayer = L.Layer.extend({
 	},
 
 	pauseDrawing: function () {
-		if (this._painter && app.sectionContainer)
+		if (this._painter && app.sectionContainer) {
 			app.sectionContainer.pauseDrawing();
+			++this._drawingPaused;
+		}
 	},
 
 	resumeDrawing: function (topLevel) {
-		if (this._painter && app.sectionContainer)
+		if (this._painter && app.sectionContainer) {
 			app.sectionContainer.resumeDrawing(topLevel);
+			if (this._drawingPaused && --this._drawingPaused === 0)
+				app.sectionContainer.requestReDraw();
+		}
 	},
```

Comment:
This tripped me up a bit - CanvasSectionContainer has odd behaviour with pausing and resuming. It seems like it's intended to function as this code does, but in practice doesn't. I don't think the requestReDraw here ought to be necessary, but it is.

For whatever reason, at this point on resumeDrawing, the canvas isn't marked as dirty and so the redraw doesn't happen. There could be something I'm not understanding there, or maybe it could be the source of other missing redraw bugs?